spec/controllers/v0/benefits_claims_controller_spec.rb

```diff
@@ -143,6 +143,10 @@
                   claim_type_code: '020NEW',
```

Comment:
@samcoforma what are you getting these values from? Im not seeing that you had to update any mock data that you changed here

Reply:
@pmclaren19 from `spec/support/vcr_cassettes/lighthouse/benefits_claims/show/200_response.yml`

Reply:
I didn't need to change the mock data, I just needed to start logging what was already there...

Reply:
Okay that seems to match up!